traitcuration/traits/datasources/ols.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+BASE_URL = "https://www.ebi.ac.uk/ols/api/"
+
+
+def make_ols_query(term_uri, ontology):
+    """
+    Takes in a term URI (or IRI as referenced in OLS documentation) and the ontology id to search against. Returns a
```

Comment:
I've finally taken some time to research the different abbreviations. Looks like IRI is the most inclusive term, which is basically a URI which can have _any_ characters in it, not just ASCI, and it does not have to have percent encoding.

So I suggest we change all occurrences of "uri" to "iri", and it would also help to include a short description of what a IRI is.

Reply:
Essentially (if I've got that right),

IRI = https://en.wiktionary.org/wiki/tréma
URI = https://en.wiktionary.org/wiki/tr%C3%A9ma (**must** be percent encoded)

Reply:
Sure, I'll change that in the data model as well